auto-generated/alias_key.json

```diff
@@ -0,0 +1,4 @@
+{
+  "A": "A",
```

Comment:
Does this line make sense? It is different than in [pango-designations](https://github.com/cov-lineages/pango-designation/blob/master/pango_designation/alias_key.json) and might cause a resolver implemented for Pango to end up in an infinite loop.

Reply:
Fair point - I'll stick t opango conventions then :)